traefik/traefik.toml

```diff
@@ -0,0 +1,36 @@
+defaultEntryPoints = ["web", "web-secure"]
+
+[log]
+  level = "DEBUG"
+  filePath = "/proc/self/fd/1"
+  format = "json"
+
+[api]
+  dashboard = true
+  insecure = true
+
+[providers.docker]
+  watch = true
+  exposedByDefault = false
+  network = "web"
+
+[providers.file]
+  filename = "/etc/traefik/dynamic_conf.toml"
+  watch = true
+
+[entryPoints]
+  [entryPoints.web]
+    address = ":80"
+  [entryPoints.web-secure]
+    address = ":443"
+
+[entryPoints.traefik]
+  address = ":8080"
+
+
+[certificatesResolvers.sample.acme]
+  email = "yoan.blanc@he-arc.ch"
```

Comment:
ça ne fonctionne plus depuis longtemps ça :wink:

Reply:
By following the new guidelines for Traefik V2,  they say that defining the routers and services at the container level is the good practice as I understood it https://docs.traefik.io/migration/v1-to-v2/

Reply:
Oh I'm sorry. I was reading on mobile yesterday evening and the lines were showing the entrypoints! I don't know about the email address, this was already on the file ;)